content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -162,6 +162,7 @@ different Kubernetes components.
 | `ValidateProxyRedirects` | `true` | Beta | 1.14 | |
 | `VolumeSnapshotDataSource` | `false` | Alpha | 1.12 | 1.16 |
 | `VolumeSnapshotDataSource` | `true` | Beta | 1.17 | - |
+| `VolumePVCDataSource` | `true` | GA | 1.20 | - |
```

Comment:
This is in the wrong section, I'm afraid - it needs to be in the section titled “Feature gates for graduated or deprecated features”.

Reply:
It is weird we didn't document `VolumePVCDataSource` before.

Reply:
Got it.  Moved to the right table.